extensions/redis-client/deployment/src/main/java/io/quarkus/redis/client/deployment/DevServicesRedisProcessor.java

```diff
@@ -138,6 +140,15 @@ private StartResult startContainer(String connectionName, DevServicesConfig devS
             return null;
         }
 
+        if (dockerRunning == null) {
+            dockerRunning = new IsDockerRunningSilent().getAsBoolean();
+        }
+
+        if (!dockerRunning) {
+            log.warn("Docker isn't working, please configure the Redis server.");
+            return null;
+        }
```

Comment:
Is it something we do in the other dev services?

Reply:
Good question. I've seen it for Mongo and Kafka but the logs are different.
- https://github.com/quarkusio/quarkus/blob/main/extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/DevServicesMongoProcessor.java#L134
-  https://github.com/quarkusio/quarkus/blob/main/extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/DevServicesKafkaProcessor.java#L205